Libraries/polyfills/performance.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ * @polyfill
+ */
+
+if (!global.performance) {
+  global.performance = {
+    now: function() {
+      return global.nativePerformanceNow();
```

Comment:
Is it possible for `nativePerformanceNow` to not exist after this change? If so, it'd be nice to fall back to the less-precise method in that case.